tests/test_numeric/test_eulerian_grid_ops/test_poisson_solver_2d/test_fft_mpi_2d.py

```diff
@@ -9,7 +9,7 @@
 from scipy.fft import rfftn
 
 
-@pytest.mark.mpi(group="MPI_Poisson_solver_2d")
+@pytest.mark.mpi(group="MPI_Poisson_solver_2d", min_size=2)
```

Comment:
Please don't make unrelated changes in a PR.

Reply:
Will take note of this in the future.